spec/features/users/password_recovery_via_recovery_code_spec.rb

```diff
@@ -24,6 +25,36 @@
     expect(page).to have_content t('headings.account.verified_account')
   end
 
+  scenario 'resets password and reactivates profile with no personal key', email: true, js: true do
+    personal_key_from_pii(user, pii)
+    trigger_reset_password_and_click_email_link(user.email)
+    reset_password_and_sign_back_in(user, new_password)
+    click_submit_default
+    enter_correct_otp_code_for_user(user)
+    visit account_path
+
+    expect(page).not_to have_content(t('headings.account.verified_account'))
+
+    visit reactivate_account_path
+    click_on t('links.account.reactivate.without_key')
+    click_on t('forms.buttons.continue')
+
+    click_idv_begin
+    fill_out_idv_form_ok
+    click_idv_continue
+    fill_out_financial_form_ok
+    click_idv_continue
+    click_idv_address_choose_phone
+    fill_out_phone_form_ok(user.phone)
+    click_idv_continue
+    fill_in 'Password', with: new_password
+    click_submit_default
```

Comment:
What do you think about using the helper method `complete_idv_profile_ok(user)`, which performs lines 43-51?